xmake/plugins/project/clang/compile_commands.lua

```diff
@@ -44,8 +44,10 @@ function _translate_arguments(arguments)
         if idx == 1 and is_host("windows") and path.extension(arg) == "" then
             arg = arg .. ".exe"
         end
-        if arg:find("-isystem", 1, true) then
-            arg = arg:replace("-isystem", "-I")
+        if arg:startswith("-isystem-after", 1, true) then
+            arg = "-I" .. arg:sub(15, -1)
```

Comment:
we just write `arg:sub(15)`

Reply:
Oh... Yes it is.
I tried that before, I had an error. I think I wrote `arg.sub` instead of `arg:sub` when I got the error and then use `arg:sub` when I commit. Let me correct that 👍

Reply:
Resolved at [c7d932e](https://github.com/xmake-io/xmake/pull/2668/commits/c7d932ea12a0fa61951c1fd9f6d87e2674aa0569)